src/models/topic-extraction/TopicExtractionJobOptions.ts

```diff
@@ -0,0 +1,14 @@
+import { RevAiApiTranscript } from "../RevAiApiTranscript";
+
+/**
+ * Options that can used when submitting Rev AI topic extraction job.
+ * See https://docs.rev.ai/api/topic-extraction/reference/#operation/SubmitTopicExtractionJob for more details.
+ */
+export interface TopicExtractionJobOptions {
+    metadata?: string;
+    callbackUrl?: string;
```

Comment:
are you sure we are getting camelCase rather than snake case responses?

Reply:
yeah, these should be snake case